Disambiguated text.txt

```diff
@@ -0,0 +1,1582 @@
+"<different>"
+	"different" adj
+"<jobs>"
+	"job" n pl
+"<where>"
+	"where" adv
+;	"where" adv itg
+"<automation>"
+	"automation" n sg
+"<is>"
+	"be" vbser pres p3 sg
+"<useful>"
+	"useful" adj
+"<.>"
+	"." sent
+"<Some>"
+;	"some" adj
+;	"some" adv
+	"some" det qnt sp
+;	"some" prn ind mf pl
+"<examples>"
+	"example" n pl
+"<are>"
+	"be" vbser pres
+"<controlling>"
+;	"controlling" n sg
+	"control" vblex pprs
+;	"control" vblex subs
+;	"control" vblex ger
+"<traffic lights>"
+	"traffic light" n pl
+"<,>"
+	"," cm
+"<vehicle>"
+	"vehicle" n sg
+"<computers>"
+	"computer" n pl
+"<,>"
+	"," cm
+"<security>"
+	"security" n sg
+"<systems>"
+	"system" n pl
+"<,>"
+	"," cm
+"<washing machines>"
+	"washing machine" n pl
+"<and>"
+	"and" cnjcoo
+"<digital>"
+	"digital" adj
+"<televisions>"
+	"television" n pl
+"<..>"
+	".." sent
+
+"<Computers>"
+	"computer" n pl
+"<can>"
+;	"can" n sg
+;	"can" vbmod inf
+	"can" vbmod pres
+;	"can" vblex inf
+;	"can" vblex pres
+;	"can" vblex imp
+"<be>"
+	"be" vbser inf
+"<designed>"
+	"design" vblex pp
+;	"design" vblex past
+;	"designed" adj
+"<to>"
+	"to" pr
+"<do>"
+;	"do" vblex inf
+;	"do" vblex pres
+;	"do" vblex imp
+;	"do" vaux inf
+;	"do" vaux pres
+	"do" vbdo inf
+;	"do" vbdo pres
+;	"do" vbdo imp
+"<almost>"
+	"almost" adv
+"<anything>"
+	"anything" prn ind mf sg
+"<with>"
+	"with" pr
+"<information>"
+	"information" n sg
+"<.>"
+	"." sent
+"<Computers>"
+	"computer" n pl
+"<are>"
+	"be" vbser pres
+"<used to>"
+	"use# to" vblex past
+	"use# to" vblex pp
+"<control>"
+;	"control" n sg
+	"control" vblex inf
+;	"control" vblex pres
+;	"control" vblex imp
+"<large>"
+	"large" adj sint
+"<and>"
+	"and" cnjcoo
+"<small>"
+	"small" adj sint
+"<machines>"
+	"machine" n pl
+"<which>"
+;	"which" det itg sp
+;	"which" prn itg mf sp
+	"which" prn rel an mf sp
+"<in the past>"
+	"in the past" adv
+"<were>"
+	"be" vbser past
+"<controlled>"
+;	"control" vblex past
+	"control" vblex pp
+"<by>"
+;	"by" adv
+	"by" pr
+"<humans>"
+	"human" n pl
+"<.>"
+	"." sent
+"<Most>"
+;	"most" adv
+;	"most" preadv
+	"most" det qnt sp
+;	"most" prn ind mf pl
+"<people>"
+;	"people" n sg
+	"person" n pl
+"<have>"
+;	"have" vbhaver inf
+	"have" vbhaver pres
+;	"have" vblex inf
+;	"have" vblex pres
+;	"have" vblex imp
+"<used>"
+;	"use" vblex past
+	"use" vblex pp
+"<a>"
+	"a" det ind sg
+"<personal>"
+	"personal" adj
+"<computer>"
+;	"computer" adj
+	"computer" n sg
+"<in>"
+;	"in" adv
+	"in" pr
+"<their>"
+;	"their" det pos sp
+	"prpers" det pos p3 mf pl
+"<home>"
+;	"home" adv
+	"home" n sg
+"<or>"
+	"or" cnjcoo
+"<at work>"
+	"at work" adv
+"<.>"
+	"." sent
+"<They>"
+	"prpers" prn subj p3 mf pl
+"<are>"
+	"be" vbser pres
+"<used>"
+;	"use" vblex past
+	"use" vblex pp
+"<for>"
+;	"for" cnjadv
+	"for" pr
+"<things>"
+	"thing" n pl
+"<such as>"
+;	"such as" adv
+	"such as" pr
+"<calculation>"
+	"calculation" n sg
+"<,>"
+	"," cm
+"<listening to>"
+	"listen# to" vblex pprs
+;	"listen# to" vblex subs
+;	"listen# to" vblex ger
```

Comment:
Reread the tagging guidelines on -ing: http://wiki.apertium.org/wiki/Tagging_guidelines_for_English#Verbs_with_.22-ing.22